scripts/container-tests

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+get_es_stats() {
+  echo 'Waiting for ES to start'
+  until curl --output /dev/null --silent --head --fail http://es:9200; do
+      printf '.'
+      sleep 5
+  done
+  echo "ES has started"
+
+  echo "Waiting for up to 60 seconds for the cluster status be at least yellow"
+  curl --silent --output /dev/null "http://es:9200/_cluster/health?wait_for_status=yellow&timeout=60s"
+  curl --silent "http://es:9200/_nodes/stats" > es-stats.json
+  echo "Saved http://es:9200/_nodes/stats to es.stats.json"
+  exit_status=0
+}
+
+validate_json() {
+  FILE=$1
+  echo "Validating $FILE"
+  if ! jq '. | empty' "$FILE"; then
+    echo "Unable to parse $FILE" >&2
+    exit 1
+  fi
+}
+
+test_max_file_descriptors() {
+  echo  "Test max file descriptors"
+  expected_max=1048576
+  max_file_descriptors="$(jq '.. | .process?.max_file_descriptors? | select(. != null)' es-stats.json)"
+
+  if [ "$max_file_descriptors" -eq $expected_max ]
+  then
+    echo "Max file descriptors test passed. Expected $expected_max, got $max_file_descriptors"
+  else
+    exit_status=1
+    echo "Max file descriptors test failed. Expected $expected_max, got $max_file_descriptors"
+  fi
+}
+
+test_number_of_docs() {
+  echo "Test number of docs in index"
+  expected_docs=9537
+  number_of_docs=$(curl -s "http://es:9200/profiles/_count?q=*" | jq '.count')
```

Comment:
Changed this from the inspection of the es-stats.json file as the introduction of nested fields introduces hidden documents as a consequence of the way ES indexes objects which in turn increases the document count in the stats. See [here](https://www.elastic.co/guide/en/elasticsearch/guide/current/nested-objects.html) for more detail.